plugins/BulkTracking/Tracker/Handler.php

```diff
@@ -37,6 +38,22 @@ public function onAllRequestsTracked(Tracker $tracker, RequestSet $requestSet)
         // Do not run schedule task if we are importing logs or doing custom tracking (as it could slow down)
     }
 
+    public function process(Tracker $tracker, RequestSet $requestSet)
+    {
+        $skippedRequests = 0;
+        foreach ($requestSet->getRequests() as $request) {
+            try {
+                $tracker->trackRequest($request);
+            } catch (UnexpectedWebsiteFoundException $ex) {
+                $skippedRequests += 1;
+            }
+        }
+
+        /** @var Response $response */
+        $response = $this->getResponse();
+        $response->setSkippedCount($skippedRequests);
```

Comment:
I was wondering why we set the number of skipped requests in `Response`? We do set the number of logged requests in `$tracker->getCountOfLoggedRequests()`. I presume it is because only BulkTracking has currently skipped requests and as we have a custom `BulkTracking\Response` that seems valid :+1: 

Just a random thought: I was wondering if the name `invalid` would be better than `skipped` but ignore it if not.